### PR TITLE
(MODULES-4726) Fix-ups for foss acceptance testing

### PR DESCRIPTION
**Pull request by johnduarte**

### CODE REVIEW

File: spec/acceptance/help_spec.rb [hunk 1]
```diff
@@ -1,26 +1,47 @@
 require 'spec_helper_acceptance'
 
-describe "C98923 - Verify that 'puppet certregen --help' prints help text" do
-    describe command("puppet certregen --help") do
-      its(:stdout) { should match( /.*USAGE: puppet certregen <action>.*/ ) }
-      its(:stdout) { should match( /.*See 'puppet man certregen' or 'man puppet-certregen' for full help.*/ ) }
+describe "pupper help certregen" do
+  # NOTE: MODULES-4733 certregen is not currently compatible with ruby < 1.9
+  ruby_ver = 0
```
Comment (adrienthebo) [inline spec/acceptance/help_spec.rb]: These changes add a lot of logic that's performed in place; for the sake of maintainability these should be extracted into helper functions and placed in either a module or an RSpec shared context. This isn't a blocker but if this code is touched soon it should be refactored out.

File: spec/acceptance/workflow_regen_after_expire_spec.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 # https://forge.puppet.com/puppetlabs/certregen#revive-a-ca-thats-already-expired
 describe "C99821 - workflow - regen CA after it expires" do
-  if hosts_with_role(hosts, 'master').length>0 and hosts_with_role(hosts, 'database').length>0 then
+  if find_install_type == 'pe' then
```
Comment (adrienthebo) [inline spec/acceptance/workflow_regen_after_expire_spec.rb]: Do PE tests imply that these roles are present, and do FOSS tests never exercise this code? The new code seems to be a bit orthogonal with respect to the old code.

Comment (johnduarte) [inline spec/acceptance/workflow_regen_after_expire_spec.rb]: 1. The 'pe' install type triggers a PE installation which defaults to a monolithic master install (including master, and database). See: https://github.com/puppetlabs/beaker-puppet_install_helper/blob/master/lib/beaker/puppet_install_helper.rb#L47-L50 and https://github.com/puppetlabs/beaker-pe/blob/master/lib/beaker-pe/install/pe_utils.rb
2. This would prevent the FOSS tests from exercising this code. Such an exercise would not pass since the puppetdb query in the gem is specific to PE. See: https://github.com/puppetlabs/puppetlabs-certregen/blob/master/lib/puppet_x/certregen/ca.rb#L114

File: spec/acceptance/help_spec.rb (hunk 1 above)
Comment (johnduarte) [inline spec/acceptance/help_spec.rb]: Thanks @adrienthebo . There is a Beaker helper that can simplify these checks. I have refactored the code to utilize it.
